### PR TITLE
Demo update

### PR DESCRIPTION
**Pull request by cpaxton**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

File: projects/spot/offline_mapping.py [hunk 1]
```diff
@@ -293,7 +293,12 @@ def get_semantic_map_vis(
                 continue
             if instance_memory is not None:
                 # retrieve name
-                category = instance_memory.instance_views[0][int(instance)].category_id
+                try:
+                    category = instance_memory.instance_views[0][
+                        int(instance)
+                    ].category_id
+                except KeyError as e:
+                    breakpoint()
```
Comment (SergioArnaud) [inline projects/spot/offline_mapping.py:301]: Should the exception be handled in another way?

File: src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py [hunk 2]
```diff
@@ -337,10 +337,13 @@ def add_view_to_instance(
             global_instance.instance_views.append(instance_view)
         self.local_id_to_global_id_map[env_id][local_instance_id] = global_instance_id
         if self.debug_visualize:
+            import os
+
```
Comment (SergioArnaud) [inline src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py:341]: `os` gets imported locally so this shouldn't be necessary

**Review by SergioArnaud — COMMENTED** [review]
Other than two minor comments this should be alright. Hopefully @Jdvakil  and I have some spot-time tomorrow to test this code with the robot and approve the PR.

File: src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py (hunk 2 above)
Comment (cpaxton) [reply src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py:341]: For some reason it is, @yvsriram any idea why?

Comment (yvsriram) [reply src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py:341]: Not sure, checking now.

Comment (yvsriram) [reply src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py]: Seems to work fine for me. I removed these. @cpaxton can you retry once again if you get a chance? But I think it should be fine.

**Review by yvsriram — APPROVED** [review]
LGTM

**Review by SergioArnaud — APPROVED** [review]
This works, let's merge it and work in demo now

**Review by Jdvakil — APPROVED** [review]
LGTM, tested on the spot and it explores

**Pull request merged**